releasenotes/README.md

```diff
@@ -12,10 +12,16 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 [Click here for the v50 release notes.](./v50.md)
 
-## March 4, 2021
+## March 10, 2021
 
 These changes are in the release candidate plug-in (`salesforcedx@latest-rc`). We plan to include these changes in the next official release. This list isn't final and is subject to change.
 
+* FIX: When running `force:mdapi:deploy` with the `runtests` flag, the specified tests will now be ran.
+* FIX: If a `force:mdapi:deploy` with the `json` flag failed, the exit code was 0, it is now 1.
```

Comment:
Edit: "When you run the `force:mdapi:deploy` command with the `--json` parameter and the command fails, it returns the exit code 1.  Previously it returned 0."